pkg/asset/ignition/machine/node.go

```diff
@@ -36,6 +37,8 @@ func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, ro
 		if installConfig.VSphere.APIVIP != "" {
 			ignitionHost = net.JoinHostPort(installConfig.VSphere.APIVIP, "22623")
 		}
+	case kubevirttypes.Name:
+		ignitionHost = net.JoinHostPort(installConfig.Kubevirt.APIVIP, "22623")
```

Comment:
I think that this section could be refactored: 
`switch installConfig.Platform.Name() {
case baremetaltypes.Name:
	apiVipParam = installConfig.BareMetal.APIVIP
....
}
ignitionHost = net.JoinHostPort( apiVipParam , "22623")
`


But I'm not sure